python/src/iceberg/expressions/base.py

```diff
@@ -122,3 +125,32 @@ def __le__(self, other):
 
     def __ge__(self, other):
         return self.value >= other.value
+
+
+class Accessor:
+    """An accessor for a specific position in a container that implements the StructProtocol"""
+
+    def __init__(self, position: int, iceberg_type: IcebergType):
+        self._position = position
+        self._iceberg_type = iceberg_type
```

Comment:
Yeah, I didn't notice this before, but I'd probably remove the type. No need for it.

Reply:
Removed! Thanks